weedcoco/tests/test_validation.py

```diff
@@ -276,3 +280,15 @@ def test_coco_compatible_good(func, coco):
 def test_coco_compatible_bad(func, bad_coco):
     with pytest.raises(JsonValidationError):
         func(bad_coco, schema="compatible-coco")
+
+
+def test_pycocotools_quirks():
+    weedcoco = copy.deepcopy(BBOX_ONLY_WEEDCOCO)
```

Comment:
I would probably artificially alter another test dataset, rather than adding a new one, and maybe considering altering only one annotation of many. It makes your test condition a bit clearer, and won't require changes to the test data to match future specs.

Reply:
This one is fun - I tried changing it to use SMALL_WEEDCOCO and found that the latter is making pycocotools throw a different exception even when it's unmodified.

Reply:
SMALL_WEEDCOCO was making pycocotools throwing an exception because a couple of the segmentations had an odd number of values - I've amended these and redone the above test to use an artificially modified version of SMALL_WEEDCOCO

Reply:
yes, https://github.com/Weed-AI/Weed-AI/pull/90 tests some such things, but I've not had much motivation to finish it with tests... Testing by comparison to pycocotools might also be an option!